content/docs/commands/list.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: 指令清單
+description: 要查詢最新的指令訊息可以使用 &help 指令
```

Comment:
Possible misleading information, more review needed

Reply:
maybe change to `/help`?

Reply:
bro then you got to re-new the entire docs because there's lots of outdated prefix commands mentioned in the docs

Reply:
Maybe we can remove the docs of command list and indeed tell them to use Discord's new command interface

Reply:
> Maybe we can remove the docs of command list and instead tell them to use Discord's new command interface

ahh that's a good idea, I'll do it in a separate PR.